.env.example

```diff
@@ -0,0 +1,34 @@
+# RPC
+RPC_URL=wss://mainnet.gateway.tenderly.co
+
+################################
+# Keeper
+################################
+# Private key. Only required when running the `drip` script
+PRIVATE_KEY=
+
+
+################################
+# Deploy
+################################
+# Ethereum address receiving the fees (recipient of CoW order)
+RECEIVER=0xA03be496e67Ec29bC62F01a428683D7F9c204930
+
+# ERC20 address of the buyToken in which you want to convert the fees into
+TO_TOKEN=0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
```

Comment:
On the assumption that the defaults are to be used for multi-chain deployment, and all other addresses are consistent across chain, I'd recommend _not_ setting a default wrapped native token here to prevent foot-guns when deploying to other chains and forgetting to change the `TO_TOKEN`.

Reply:
Similar point for `MIN_OUT` in the event that the native token is _not_ ETH (and whereby the `MIN_OUT` would actually be chain-specific).

Reply:
killed most of the defaults. I actually did want to add any :)